### PR TITLE
Rename (mv) folders and files. Refs #1044

### PR DESCRIPTION
**Pull request by OSdave**

### CODE REVIEW

**Review by OSdave — COMMENTED** [review]
files viewed

File: routers/bucket/actions/index.js [hunk 1]
```diff
@@ -62,14 +63,17 @@ module.exports = ({actions, storage, ...config}) => {
         res.send(204);
       })
       .catch((err) => {
-        res.send(err.message);
```
Comment (llambeau) [inline routers/bucket/actions/index.js:65]: if (err instanceof ....) {
  // handle case for this class, let's say it's not found => 404
} else if (err instanceof ....) {
  // other error, other err code
}

File: webspicy/actions/mv.yml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    description: |-
+      Allows renaming an existing directory or file
+
+    preconditions: |-
+      - The directory simplest/old should exist
+      - The file simplest/old.txt should exist
+
+    postconditions: |-
+      - The directory simplest/old must not exist
```
Comment (llambeau) [inline webspicy/actions/mv.yml:20]: suggestion: split this one test suite in two test files:

- one for the mv of files
- one for the mv of folders

Each of them only including the pre/post conditions that are relevant to the use case

**Review by llambeau — APPROVED** [review]
After split of test suites: you can merge

**Comment by llambeau** [comment]
Closed, this PR is now part of https://github.com/enspirit/seshat/pull/21/commits

**Pull request closed without merging**